Queue.py

```diff
@@ -0,0 +1,80 @@
+#Creamos La claes cola que tendra todas las funciones
+class Queue:
+    def __init__(self): # Fixed typo: _init __ should be __init__
+        self.items = []
+
+    def enqueue(self, item):
+        self.items.append(item)
+
+    def dequeue(self):
+        if self.is_empty():
+            raise IndexError("La cola esta vacía")
+        return self.items.pop(0) # Elimina y devuelve el primer elemento
+
+    def front(self):
+        if self.is_empty():
+            raise IndexError("La cola esta vacía")
+        return self.items[0] # Devuelve el primer elemento sin eliminarlo
+
+    def rear(self):
+        if self.is_empty():
+            raise IndexError("La cola esta vacía")
+        return self.items[-1] # Devuelve el ultimo elemento sin eliminarlo
+
+    def is_empty(self):
+        return len(self.items) == 0 # Verifica si La cola está vacía
+
+    def size(self):
+        return len(self.items)
+
+    def clear(self):
+        self.items.clear()
+
+    def peek(self, index):
```

Comment:
No tienen una operación para hacer lectura, no te preocupes, yo te ayudo.
```suggestion
    def peek(self, index):
        if index < 0 or index >= len(self.items):
            raise IndexError("Indice fuera de rango")
        return self.items[index] 
```